ra2ce/analysis/analysis_config_data/analysis_config_data.py

```diff
@@ -98,7 +98,6 @@ class AnalysisSectionLosses(AnalysisSectionBase):
     production_loss_per_capita_per_day: float = math.nan
     part_of_day: PartOfDayEnum = field(default_factory=lambda: PartOfDayEnum.DAY)
     performance: str = "diff_time"  # "diff_time" or "diff_length" relates to the used criticality metric
```

Comment:
```suggestion
    performance: str = "diff_time"  # "diff_time" or "diff_length" relates to the used criticality metric
    hours_per_day: float = 24
```

Reply:
I suggest the opposite to keep things as simple and unambiguous as possible. When we feel ready to have this feature and we figured out the math behind it, we will reintegrate it in the code.

Reply:
Plus, we still need to use it with the default value of 24h.